OpenOversight/app/templates/list_officer.html

```diff
@@ -160,6 +160,10 @@ <h2>
                       <dd>{{ officer.unit_descrip()|default('Unknown') }}</dd>
                       <dt>Currently on the Force</dt>
                       <dd>{{ officer.currently_on_force() }}</dd>
+                      {% if officer.incidents %}
+                      <dt>Known incidents</dt>
+                      <dd>{{ officer.incidents | length }}</dd>
+                      {% endif %}
```

Comment:
From a UX perspective, I feel like it might be confusing to users if this field is shown for some officers and not others. What do you think?

Reply:
That's a fair point! My concern is that, if we have the value for cases where WE don't have any incidents, users might take that to mean there _are no incidents at all_ for this officer. I don't want to give that impression because there is certainly more than that (like Use of Force which we aren't even counting). What do you think? Which setup is less confusing?

Reply:
I think we run into the same issue if we say there's 1 incident for that officer, since there could be more we haven't added to the database, right? Maybe we could change the wording from "Known incidents" to something like "Incidents in database"? Definitely not a blocker so feel free to resolve if there's not a better wording

Reply:
That's a fair point! Okay I'll remove the conditional logic, and I think just leave the wording as-is. Thanks!